### PR TITLE
Milestone 4: Desktop version

### PR DESCRIPTION
**Pull request by WilfriedM01**
**Here are the functionnalities**

- Used media query to manage different devices
- Used Flexbox and Grid to layout elements
- Took care to use best practices by coding

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -356,14 +365,40 @@ input {
...
+.article-btn:active {
+  background-color: #5280f5;
+  color: #fff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+}
+
+.article-item:hover {
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+}
+
```
Comment (Ibizugbe) [inline assets/css/style.css:401]: Nice job so far implementing the button interactions for each of your buttons😉👍. I noticed you must have forgotten to add the button interactions for your `get in touch` button😉. Kindly do well to implement it👍.
image

File: index.html [hunk 2]
```diff
@@ -81,13 +81,13 @@ <h3>CONOPY <span>&bull; Back End Dev &bull; 2015</span></h3>
...
-          </artcle>
+          </article>
         </section></a
       >
       <a href="#" class="article-link"
         ><section class="articles">
-          <artcle class="article-item">
-            <p>
+          <article class="article-item item-reverse">
+            <p class="sub-item">
```
Comment (Ibizugbe) [inline index.html:90]: Nice job making your project look responsive, everything looks great😉. Just this little issue on the mobile screen for each of the project cards😉. Take note of the paddings around card and make it match the given design👍.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|image|image|

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|image|image|

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @WilfriedM01 👋👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Linters are passing👍.
- [x] Github flow is used👍.
- [x] Media queries for `768px` is used👍. 
- [x] Descriptive commit messages👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (WilfriedM01) [reply index.html:90]: Hi @Ibizugbe 

Thank you for reviewing, now I have improved the UI and UX :+1:

**Review by skyv26 — APPROVED** [review]
## Hi @WilfriedM01, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Linters are green ✅ 
- Meaningful **Github** commits ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**